src/core/util/generators/uiSchema.ts

```diff
@@ -5,16 +5,27 @@
  * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
  * ---------------------------------------------------------------------
  */
-import { ControlElement, Layout } from '@jsonforms/core';
+import { v4 as uuid } from 'uuid';
 
-export const createControl = (scope: string): ControlElement => {
-  return {
+import { getPath, SchemaElement } from '../../model';
+import { LinkedUISchemaElement } from '../../model/uischema';
+export const createControl = (
+  schemaElement: SchemaElement
+): LinkedUISchemaElement => {
+  const scope = `#${getPath(schemaElement)}`;
+  const newControl = {
     type: 'Control',
     scope: scope,
+    uuid: uuid(),
   };
+  return newControl;
 };
 
-export const createLayout = (type: string): Layout => ({
-  type: type,
-  elements: [],
-});
+export const createLayout = (type: string): LinkedUISchemaElement => {
+  const newLayout = {
```

Comment:
why not directly return?

Reply:
We can return here directly, I'll fix in a new commit.